crates/matrix-sdk-ui/src/timeline/event_handler.rs

```diff
@@ -921,6 +927,7 @@ impl<'a, 'o> TimelineEventHandler<'a, 'o> {
                 reactions: Default::default(),
                 thread_root: None,
                 in_reply_to: None,
+                thread_summary: None,
```

Comment:
Can't poll be part of a thread or thread roots? (if they can: let's also add a TODO here)

Reply:
To make things fun polls can indeed be thread roots but they can't be sent inside threads 🤪